app/views/places/index.html.erb

```diff
@@ -20,10 +20,15 @@
         <p><%= link_to t('.destroy'), place, method: :delete, data: { confirm: t('.sure') } %></p>
       </div>
     <% end %>
-    <%= link_to t('.newplace'), new_place_path %>
+    <br>
+    <%= link_to t('.newplace'), new_place_path, :class => "btn btn-secondary btn-lg btn-block" %>
```

Comment:
Please use format: `class: 'btn btn-secondary btn-lg btn-block'` indead of `:class => "btn btn-secondary btn-lg btn-block"` It looks cleaner.